### ISSUE TITLE
Wolfram Removes "Who is" and "Who was" from Query

### ISSUE THREAD

**Issue by ryanleesipes** [MycroftAI/mycroft-core#63]
Had Wikipedia handling this, but Wolfram should handle it better, need to fix it so that "Who is" and "Who was" are not removed from the query.

**Comment by ryanleesipes** [comment]
To elaborate all question type queries have the question parts removed. This makes the Wolfram skill nearly completely useless.

**Comment by ryanleesipes** [comment]
Reassigned to @jdorleans

**Comment by ryanleesipes** [comment]
This should be fixed by #77

**Comment by ryanleesipes** [comment]
Perhaps this is a problem related to #79 with the queries that are failing.

**Comment by ryanleesipes** [comment]
Many queries are still not answerable through our Wolfram Alpha skill, that are answerable through the [web frontend](http://www.wolframalpha.com/). We are doing something wrong in processing these. Reassigned to @ethanaward

**Comment by ethanaward** [comment]
I've isolated this problem to the Wolfram Alpha skill. If I ask Mycroft `who was Abraham Lincoln`, we get the information from Wolfram about who Abraham Lincoln was, but the skill then looks for a "pod" or chunk of information called `Result`, which a search for a person doesn't give you.  

One possible fix would be to have it look for `Result`, then if that doesn't exist, a pod title that all people queries have, and so on. We would want to try to provide complete coverage though, which may be tricky. 

While looking for a way to do this, I've noticed that the titles all of the pods in a person query have `PeopleData` in them, e.g. `id='WikipediaSummary:PeopleData'`. This means we could possibly search through the titles of the response to determine what data we should extract from the response. This could potentially not work if the API changes, however.
